packages/shared/routes/setup/migrate/views/Migrate.svelte

```diff
@@ -91,16 +97,11 @@
                             closePopup(true) // close transaction popup
                             singleMigrationBundleHash = bundleHash
                             const reverseTrytesLedger = trytes.reverse()
-                            prepareMigrationLog(bundleHash, reverseTrytesLedger, migratableBalance)
+                            prepareMigrationLog(reverseTrytesLedger, migratableBalance, bundleHash)
                             return sendOffLedgerMigrationRequest(reverseTrytesLedger, 0)
                         })
                         .then((receipt) => {
-                            migrationLog.update((_migrationLog) =>
-                                _migrationLog.map((log) => ({
-                                    ...log,
-                                    requestId: receipt?.request?.requestId || '',
-                                }))
-                            )
+                            updateRequestInMigrationLog(receipt?.request, 0)
```

Comment:
If user retries the migration new log is added on this line `prepareMigrationLog(reverseTrytesLedger, migratableBalance, bundleHash)` but  `updateRequestInMigrationLog(receipt?.request, 0)` always updates the errorMessage for the  first log (index=0)